src/content/post/2025-08-22-daikon-vim-2.md

```
@@ -0,0 +1,38 @@
+[daikon.vim #2](https://daikonvim.connpass.com/event/363680/)に参加した。
+
+中身はもくもく会である。
+
+## やったこと
+
+### アイコンのpngをつくる
+
+VimConfのチケットを書いたいが、去年pngに変換するのが大変だったと聞いた記憶があったのでpngを用意する。
+
+```console
+$ magick convert Omochice.jpg Omochice.png
```

markdownlint-cli2 (0.17.2)

Dollar signs used before commands without showing output

(MD014, commands-show-output)